src/dashboard/components/pages/Blocks.js

```diff
@@ -211,6 +213,12 @@ const otterBlocks = [
 		'name': __( 'AI Block', 'otter-blocks' ),
 		'icon': () => aiGeneration,
 		'docLink': 'https://docs.themeisle.com/article/1917-the-ai-block-otter-blocks-library#content-generator'
+	},
+	{
+		'slug': 'themeisle-blocks/timeline',
+		'name': __( 'Timeline', 'otter-blocks' ),
+		'icon': timelineIcon,
+		'docLink': 'https://docs.themeisle.com/article/2050-modal-block'
```

Comment:
You can draft an article on the HS docs (or ask someone from Support team for help) and put that URL here.

Reply:
it's done by me. is in draft.